tenets/codelingo/code-review-comments/avoid-renaming-imports/codelingo.yaml

```diff
@@ -0,0 +1,40 @@
+funcs:
+  - name: diffNameImport
```

Comment:
Nitpicking but renaming this function to something like importNamesAreDifferent could prevent needing to add a comment here. Usually these asserter functions would be named starting with 'is' to make its function a little clearer but this has two parameters.